proprietary/tokens/src/components/ams/accordion.tokens.json

```diff
@@ -5,10 +5,10 @@
       "button": {
         "color": { "value": "{ams.color.primary-blue}" },
         "font-family": { "value": "{ams.text.font-family}" },
-        "font-size": { "value": "{ams.text.level.5.font-size}" },
+        "font-size": { "value": "{ams.text.level.4.font-size}" },
```

Comment:
We have to change the Icon size in the React component as well

Reply:
Resolving due to Aram’s absence.